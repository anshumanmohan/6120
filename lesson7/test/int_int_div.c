int main()
{
  int x = 42 / 3;
  return x;
}