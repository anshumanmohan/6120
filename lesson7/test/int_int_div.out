I saw a function called main!
